### PR TITLE
Docs whatsnew add dropdowns to the template

### PR DESCRIPTION
**Pull request by bjlittle**
### Description
This PR adds the `sphinx-panels` template `dropdown` to the `latest.rst` and the `latest.rst.template`.

---
[Consult Iris pull request check list]( https://scitools-iris.readthedocs.io/en/latest/developers_guide/contributing_pull_request_checklist.html)

### CODE REVIEW

File: docs/iris/src/whatsnew/latest.rst.template
Comment (bjlittle) [inline docs/iris/src/whatsnew/latest.rst.template:19]: This assumes that #3970 is merge :wink:

File: docs/iris/src/whatsnew/latest.rst.template
Comment (bjlittle) [reply docs/iris/src/whatsnew/latest.rst.template:10]: @tkknight Essentially, yes.

The release manager, or whom ever, for the next minor `dev` release, copies the `latest.rst.template` to the `latest.rst` and removes the patch pulldown.

If a patch release requires to be made then the release manager, or whom ever, copies the patch pulldown from the `latest.rst.template` to the appropriate release notes, which themselves will be copied from the last relevant release.

The patch release should contain both the patch pulldown and the release highlights pulldown, as per [v3.0.1](https://scitools-iris.readthedocs.io/en/stable/whatsnew/3.0.1.html), with the appropriate patch release date.

If a release has several patches, then the associated whatsnew will have several patch pulldowns, one for each patch release. We could choose to collapse all but the latest patch pulldown and the release highlights pulldown to minimise vertical footprint and draw attention to the pulldowns that matter most.

**Comment by tkknight** [comment]
Hey @bjlittle .  Happy to approve and merge this once the minor conflict is resolved.

**Review by tkknight — APPROVED** [review]

**Pull request merged**
